### PR TITLE
translation: flowcontrol plugin

### PR DESCRIPTION
**Pull request by daizhenyu**
**What type of PR is this?**

Doc

**What this PR does / why we need it?**

This initiative aims to enhance readability, accessibility, and collaboration for both current and potential contributors, ensuring a unified language throughout the codebase.

**Which issue(s) this PR fixes？**

Fixes #1466

**Does this PR introduce a user-facing change?**

No

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/huaweicloud/Sermant/issues) related with this PR before you start working on it.
- [x] Make sure you have squashed your change to one single commit.
- [x] GitHub Actions works fine in this PR.

### CODE REVIEW

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/cache/ConcurrentMapCache.java [hunk 1]
```diff
@@ -20,10 +20,10 @@
 import java.util.concurrent.ConcurrentHashMap;
 
 /**
- * 基于ConcurrentMap实现
+ * implemented based on concurrent map
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/cache/ConcurrentMapCache.java]: It is recommended that concurrent  map be changed to ConcurrentMap

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/CommonConst.java [hunk 2]
```diff
@@ -18,250 +18,254 @@
...
 
     /**
-     * dubbo服务端
+     * dubboServer
      */
     public static final String DUBBO_PROVIDER = "provider";
 
     /**
-     * 区分dubbo调用端 provider 服务端 consumer 客户端
+     * distinguish between dubbo callers: provider consumer
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/CommonConst.java]: Distinguish whether the dubbo caller is a provider server or a consumer client

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/CommonConst.java [hunk 3]
```diff
@@ -18,250 +18,254 @@
...
 
     /**
-     * 线程池启动 延迟第一次执行的时间
+     * Thread pool startup delays the time of the first execution
      */
     public static final int INITIAL_DELAY = 5000;
 
     /**
-     * 是否备份规则到redis，默认备份 ，控制台参数传费true是不备份
+     * Whether to back up rules to redis, default backup, console parameter transfer fee true is no backup
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/CommonConst.java]: There is an extra word fee

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/CommonConst.java [hunk 4]
```diff
@@ -18,250 +18,254 @@
...
     /**
-     * sentinel配置参数 查询流控数据时,睡眠一段时间，等待限流数据写入文件再查询
+     * sentinel configuration parameter, when querying flow control data, sleep for a period of time
+     * and wait until the flow limiting data is written into the file
      */
     public static final long METRIC_SLEEP_TIME = 2000L;
 
     /**
-     * kafka配置参数 控制生产者发送请求最大大小,默认1M （这个参数和Kafka主机的message.max.bytes 参数有关系）
+     * kafka, configuration parameter, Control the maximum size of requests sent by the producer. The default is 1 M
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/CommonConst.java]: It should be kafka configuration parameter. There is an extra comma.

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/ConfigConst.java [hunk 5]
```diff
@@ -18,244 +18,249 @@
...
 
     /**
-     * 加密协议常量
+     * encryption protocol constant
      */
     public static final String KAFKA_SECURITY_PROTOCOL_CONST = "security.protocol";
 
     /**
-     * 加密协议，目前支持SASL_SSL协议
+     * The encryption protocol supports the SASL SSL protocol
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/ConfigConst.java]: Protocol should be SASL_SSL

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/FlowControlConfig.java [hunk 6]
```diff
@@ -24,130 +24,134 @@
...
 
     /**
-     * SASL鉴权机制
+     * sasl authentication mechanism
      */
     private String kafkaSaslMechanism = ConfigConst.KAFKA_SASL_MECHANISM;
 
     /**
-     * 加密协议，目前支持SASL_SSL协议
+     * The encryption protocol supports the SASL SSL protocol
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/FlowControlConfig.java]: Protocol should be SASL_SSL

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/core/ResolverManager.java [hunk 7]
```diff
@@ -100,11 +100,11 @@ public void resolve(Map<String, String> rulesMap, boolean isForDelete) {
...
     /**
-     * 单个kv解析
+     * single kv analysis
      *
-     * @param key         键
-     * @param value       值
-     * @param isForDelete 是否是删除键
+     * @param key         key
+     * @param value       value
+     * @param isForDelete whether delete key
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/core/ResolverManager.java]: isForDelete  should be translated to whether it's a delete key

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/core/match/RequestMatcher.java [hunk 8]
```diff
@@ -169,11 +172,11 @@ private boolean operatorMatch(String target, RawOperator operator) {
         for (Map.Entry<String, String> entry : operator.entrySet()) {
             final Operator matchOperator = OperatorManager.INSTANCE.getOperator(entry.getKey());
             if (matchOperator == null) {
-                // 无相关匹配器，数据错误！
+                // no relevant matcher data error！
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/core/match/RequestMatcher.java]: no relevant matcher，data error

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/entity/AbstractRequestEntity.java [hunk 9]
```diff
@@ -20,19 +20,19 @@
 import java.util.Objects;
 
 /**
- * 用于统一重写equals与hashCode方法
+ * Used to uniformly override equals and hash Code methods
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/entity/AbstractRequestEntity.java]: hash Code should be change hashCode

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/entity/RequestEntity.java [hunk 10]
```diff
@@ -22,82 +22,82 @@
...
-     * @return 是否相等
+     * @param obj comparative object
+     * @return whether it is equal or not
      */
     @Override
     boolean equals(Object obj);
 
     /**
-     * 必须实现hashCode编码
+     * the hash code encoding must be implemented
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/entity/RequestEntity.java]: The hash code is a whole. You are advised to change it to hashCode

File: sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/config/SpringApplicationDeclarer.java [hunk 11]
```diff
@@ -23,7 +23,7 @@
 import com.huaweicloud.sermant.core.plugin.agent.matcher.MethodMatcher;
 
 /**
- * 增强SpringApplication类的run方法
+ * Enhance the run method of the Spring Application class
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/config/SpringApplicationDeclarer.java]: Spring Application  should be change SpringApplication

File: sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/retry/FeignRequestDeclarer.java [hunk 12]
```diff
@@ -30,21 +30,21 @@
  */
 public class FeignRequestDeclarer extends AbstractPluginDeclarer {
     /**
-     * 增强类的全限定名
```
Comment (hanbingleixue) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/retry/FeignRequestDeclarer.java:33]: Class comments are not translated

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/cache/ConcurrentMapCache.java (hunk 1 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/cache/ConcurrentMapCache.java]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/CommonConst.java (hunk 2 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/CommonConst.java]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/CommonConst.java (hunk 3 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/CommonConst.java]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/CommonConst.java (hunk 4 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/CommonConst.java]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/ConfigConst.java (hunk 5 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/ConfigConst.java]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/FlowControlConfig.java (hunk 6 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/FlowControlConfig.java]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/core/ResolverManager.java (hunk 7 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/core/ResolverManager.java]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/core/match/RequestMatcher.java (hunk 8 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/core/match/RequestMatcher.java]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/entity/AbstractRequestEntity.java (hunk 9 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/entity/AbstractRequestEntity.java]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/entity/RequestEntity.java (hunk 10 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/entity/RequestEntity.java]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/config/SpringApplicationDeclarer.java (hunk 11 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/config/SpringApplicationDeclarer.java]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/retry/FeignRequestDeclarer.java (hunk 12 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/retry/FeignRequestDeclarer.java:33]: done

File: sermant-plugins/sermant-flowcontrol/config/config.yaml [hunk 13]
```diff
@@ -1,6 +1,6 @@
-# 流控配置
+# flowControlConfiguration
```
Comment (lilai23) [inline sermant-plugins/sermant-flowcontrol/config/config.yaml]: FlowControl configuration

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/cache/ConcurrentMapCache.java [hunk 14]
```diff
@@ -20,10 +20,10 @@
 import java.util.concurrent.ConcurrentHashMap;
 
 /**
- * 基于ConcurrentMap实现
+ * implemented based on ConcurrentMap
```
Comment (lilai23) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/cache/ConcurrentMapCache.java]: ConcurrentHashMap

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/ConfigConst.java [hunk 15]
```diff
@@ -18,244 +18,249 @@
...
- * 配置常量类
+ * configuration constant class
  *
  * @author zhouss
  * @since 2022-01-28
  */
 public class ConfigConst {
     /**
-     * sentinel版本
+     * sentinel versions
```
Comment (lilai23) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/ConfigConst.java]: sentinel version

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/core/match/operator/CompareOperator.java [hunk 16]
```diff
@@ -33,7 +33,8 @@
 import java.util.logging.Logger;
 
 /**
- * 比较, 格式如下, 存在有两个符合和一个符号的场景 请求路径包含/b 且    d=10 e != 5 f >= 11 - apiPath: contains: /b headers: d: compare: =10 e:
+ * compare in the following format： There are two matches and one symbol that the scene request path contains/b and
```
Comment (lilai23) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/core/match/operator/CompareOperator.java]: There are two  scenes： two symbols and one symbol

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/entity/MetricEntity.java [hunk 17]
```diff
@@ -18,104 +18,104 @@
...
 
     /**
-     * 熔断慢调用数量
+     * Number of circuit breaker slow calls
      */
     private AtomicLong slowFuseRequest;
 
     /**
-     * 熔断调用数量
+     * number of fuse calls
```
Comment (lilai23) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/entity/MetricEntity.java]: Circuit breaker ，not fuse

File: sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/retry/handler/DefaultRetryPredicateCreator.java [hunk 18]
```diff
@@ -104,7 +104,7 @@ private boolean isGenericException(String className) {
 
     private Class<? extends Throwable> getRealExceptionClass(Throwable ex) {
         if (ex instanceof InvokerWrapperException) {
-            // 判断是否是目标包装异常
+            // Determine whether it is an exception in the target package
```
Comment (lilai23) [inline sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/retry/handler/DefaultRetryPredicateCreator.java]: wrapper not package

File: sermant-plugins/sermant-flowcontrol/config/config.yaml (hunk 13 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/config/config.yaml]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/cache/ConcurrentMapCache.java (hunk 14 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/cache/ConcurrentMapCache.java]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/ConfigConst.java (hunk 15 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/config/ConfigConst.java]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/entity/MetricEntity.java (hunk 17 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/entity/MetricEntity.java]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/retry/handler/DefaultRetryPredicateCreator.java (hunk 18 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/com/huawei/flowcontrol/retry/handler/DefaultRetryPredicateCreator.java]: done

File: sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/core/match/operator/CompareOperator.java (hunk 16 above)
Comment (daizhenyu) [reply sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/com/huawei/flowcontrol/common/core/match/operator/CompareOperator.java]: done

**Review by hanbingleixue — APPROVED** [review]

**Review by lilai23 — APPROVED** [review]

**Pull request merged**